### ISSUE TITLE
Syntax Error in   => record["message"]

### ISSUE THREAD

**Issue by Rauschii** [davidalber/geneagrapher#21]
I used the pip install and then used `ggrapher -help`
This is the message I get. 

```
Traceback (most recent call last):
  File "/anaconda3/bin/ggrapher", line 7, in <module>
    from geneagrapher.geneagrapher import ggrapher
  File "/anaconda3/lib/python3.7/site-packages/geneagrapher/geneagrapher.py", line 119
    print record["message"]
               ^
SyntaxError: Missing parentheses in call to 'print'. Did you mean print(record["message"])?
```
---
First I thought it might be because I used python3.7 but even when I switch to 2.7 it did not work. Not sure it is an issue from your code or my wrong using. Any help would be very much appreciated. Thank you!

**Comment by davidalber** [comment]
Fixed by #23. Note, however, #24, which must also be fixed before the tool works again.

**Comment by Rauschii** [comment]
thanks!
